homework/index.js

```diff
@@ -30,13 +30,149 @@
     return elem;
   }
 
+  // Load the Contributors information into  section2 div element.
+
+  function addContributors(contribution) {
+    const divElement = [];
+
+    divElement.push('<div id="header">', `${'<h3>'} Contributions: ${'</h3>'} ${'</div>'}`);
+
+    // eslint-disable-next-line no-restricted-syntax
+    for (const details of contribution) {
+      divElement.push(
+        '<ul>',
+        `${'<li>'} ${'<img src="'}${details.avatar_url}" width="40" height="40">`,
+        `</li>`,
+        `<li>${details.login}</li>`,
+        `<li>${details.contributions}</li>`,
+        '</ul>',
+      );
+    }
+
+    const htmlString = divElement.join('');
+    document.getElementById('section2').innerHTML = htmlString;
+  }
+
+  // Load the Repository information into  section1 div element.
+
+  function loadDiv(repoInfo, optionValue) {
+    const templateElement = [];
+
+    // eslint-disable-next-line no-restricted-syntax
+    for (const repo in repoInfo) {
+      if (Object.prototype.hasOwnProperty.call(repoInfo, repo)) {
+        if (optionValue === repoInfo[repo].name) {
+          // format the date
+          const upDate = new Date(repoInfo[repo].updated_at);
+          const amOrPm = upDate.getHours() < 12 ? 'AM' : 'PM';
+          const dateHours = upDate.getHours() % 12 || 12;
+          const formatedUpdate = `${upDate.getMonth()}/${upDate.getDate()}/${upDate.getFullYear()} ${dateHours}
+             : ${upDate.getMinutes()}:${upDate.getSeconds()} ${amOrPm}`;
+
+          templateElement.push(
+            '<div id="row">',
+            `${'<p id="name-info">'} Repository name : ${'<a href="'}${
+              repoInfo[repo].html_url
+            }"${'/>'} ${repoInfo[repo].name}</a></p>`,
+            `${'<p id="desc">'}  Description :  ${repoInfo[repo].description}</p>`,
+            `${'<p id="forks">'}  Forks       : ${repoInfo[repo].forks_count}</p>`,
+
+            `${'<p id="updated">'} Updated  : ${formatedUpdate}</p>`,
+            '</div>',
+          );
+
+          // Call another function(addContributors) to fill i contributor information.
+          fetchJSON(repoInfo[repo].contributors_url, (err, data) => {
+            const root = document.getElementById('root');
+
+            if (err) {
+              createAndAppend('div', root, { text: err.message, class: 'alert-error' });
+            } else {
+              addContributors(data);
+            }
+          });
+        }
+      }
+    }
+    const htmlString = templateElement.join('');
+    document.getElementById('section1').innerHTML = htmlString;
+  }
+
+  // Create options under 'SELECT' element which should have HYF Repositories.
+
+  function loadSelectionValues(userInfo) {
+    const sortRepoName = [];
+
+    const userRepo = JSON.parse(userInfo);
+    const mySelect = document.getElementById('myselect');
```

Comment:
Not sure what the word `my` helps signify. Think a better name could be used?

Reply:
corrected with name selectrepo.